features/step_definitions/networking.rb

```diff
@@ -993,10 +1006,27 @@
                                            user: admin, quiet: true) { |pod, hash|
     pod.node_name == leader_node || pod.ip == leader_node
   }.first
+  # update the cache so we can execute on the pod without specify the name
+  cache_resources leader_pod
```

Comment:
@anuragthehatter #1252 broke your other OVN tests because it removed this `cache_resources leader_pod` line.
https://github.com/openshift/verification-tests/pull/1252/files#diff-b4003f8a21fcd4d7140fd9ca8d001ac7L854

we need to add back the `cache_resources leader_pod` to fix it, so we should merge this PR soon.

Reply:
Sounds good @rbbratta. Thanks for catching that